guides/source/layout.html.erb

```diff
@@ -39,59 +51,83 @@
+              <option value="https://guides.rubyonrails.org/v3.0">3.0</option>
+              <option value="https://guides.rubyonrails.org/v2.3">2.3</option>
+            </select>
+          </span>
+        </div>
+        <ul class="nav">
+          <li><a class="nav-item" id="home_nav" href="https://rubyonrails.org/">Home</a></li>
+          <li class="guides-index guides-index-large">
+            <a href="index.html" id="guidesMenu" class="guides-index-item nav-item">Guides Index</a>
+            <div id="guides" class="clearfix" style="display: none;">
+              <hr />
+              <dl class="guides-section-container">
+                <% documents_by_section.each do |section| %>
+                  <div class="guides-section">
+                    <dt><%= section['name'] %></dt>
+                    <% finished_documents(section['documents']).each do |document| %>
+                    <dd><a href="<%= document['url'] %>"><%= document['name'] %></a></dd>
+                    <% end %>
+                  </div>
+                <% end %>
+              </dl>
+            </div>
+          </li>
+          <li><a class="nav-item" href="contributing_to_ruby_on_rails.html">Contribute</a></li>
+          <li class="guides-index guides-index-small">
+            <select class="guides-index-item nav-item">
+              <option value="index.html">Guides Index</option>
+              <% docs_for_menu.each do |section| %>
+                <optgroup label="<%= section['name'] %>">
                   <% finished_documents(section['documents']).each do |document| %>
-                  <dd><a href="<%= document['url'] %>"><%= document['name'] %></a></dd>
+                    <option value="<%= document['url'] %>"><%= document['name'] %></option>
                   <% end %>
-                </div>
+                </optgroup>
               <% end %>
-            </dl>
-          </div>
-        </li>
-        <li><a class="nav-item" href="contributing_to_ruby_on_rails.html">Contribute</a></li>
-        <li class="guides-index guides-index-small">
-          <select class="guides-index-item nav-item">
-            <option value="index.html">Guides Index</option>
-            <% docs_for_menu.each do |section| %>
-              <optgroup label="<%= section['name'] %>">
-                <% finished_documents(section['documents']).each do |document| %>
-                  <option value="<%= document['url'] %>"><%= document['name'] %></option>
-                <% end %>
-              </optgroup>
-            <% end %>
-          </select>
-        </li>
+            </select>
+          </li>
       </ul>
+      </nav>
     </div>
-  </div>
+  </header>
   <hr class="hide" />
 
-  <div id="feature">
+  <main id="feature">
     <div class="wrapper">
       <%= yield :header_section %>
 
       <%= yield :index_section %>
     </div>
-  </div>
+  </main>
 
-  <div id="container">
+  <section id="feedback">
```

Comment:
I ran into a small issue with this new ID, and I'm not sure about calling this block "feedback". Yes, it holds that last "feedback" copy at the bottom, but it's where all the main content for the guide is (where we `yield`). It just so happens that every guide ends with the Feedback paragraphs.

I have a content blocker that removes comments and a bunch of other stuff from the page, and it turns out it removed the whole content because of its `#feedback` ID. Yes, it's kind of a problem with my content blocker in Safari, as it looks okay in Chrome, but there may be others out there doing the same. We'd be okay if just the actual "feedback" paragraphs were hidden by something like this (i.e. if they were wrapped in a `#feedback` tag), not the whole content.

<details><summary>This is what it looked like, for reference</summary>
<p>

![Screenshot 2024-03-18 at 11 02 14](https://github.com/rails/rails/assets/26328/4a300d42-2d41-4007-8a05-f47e1b167827)

</p>
</details> 

Do we need the ID for styling or anything else? Can we use another ID here?

Reply:
I changed it back to `#container`